src/api/v1/permissions/routes/user.patch.ts

```diff
@@ -25,10 +25,10 @@ export default async (req: FastifyRequest<{ Params: { user: string }, Body: { pe
           id: req.params.user
         },
         data: {
-          ...user,
           permissions: req.body.permissions
         }
       })
```

Comment:
Consistent spacing pls
```suggestion
      })

```